pkg/validator/summary.go

```diff
@@ -17,6 +17,9 @@ type CountSummaryByCategory map[string]CountSummary
 // GetScore returns an overall score in [0, 100] for the CountSummary
 func (cs CountSummary) GetScore() uint {
 	total := (cs.Successes * 2) + cs.Warnings + (cs.Errors * 2)
+	if total == 0 {
```

Comment:
Thanks! I can't believe [this bug is still open](https://github.com/FairwindsOps/polaris/issues/206) 😬